src/pal/tools/gen-buildsys-gcc.sh

```diff
@@ -60,9 +62,9 @@ __UnprocessedCMakeArgs=""
 
 ITER=-1
 for i in "$@"; do
-    ITER=$(($ITER + 1))
+    ITER=$((ITER + 1))
```

Comment:
Can you please explain the purpose of the seemingly cosmetic changes in this file?

Reply:
Simplified syntax: https://github.com/koalaman/shellcheck/wiki/SC2004